.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -45,3 +45,4 @@ Examples:
 
 
 ***Is this ready for review?:*** NO
+***Is it breaking change?:*** NO
```

Comment:
`Is it breaking change?` --> `Is it a breaking change?`

And maybe add a link to the `breaking-change.md` document?